quipucords/api/messages.py

```diff
@@ -65,8 +65,7 @@
 NET_INVALID_PORT = "Source of type network must have ssh port in range [0, 65535]"
 NET_HC_DO_NOT_EXIST = "Host credential with id=%d could not be found in database."
 NET_SSL_OPTIONS_NOT_ALLOWED = "Invalid SSL options for network source: %s"
-VC_INVALID_OPTIONS = "Invalid options for vcenter source: %s"
-SAT_INVALID_OPTIONS = "Invalid options for satellite source: %s"
+INVALID_OPTIONS = "Invalid option for '%(source_type)s': %(options)s."
```

Comment:
should that be Invalid `options` since that is what is being dry'ed up ?

Reply:
whoopsie. Dunno why I removed the "s" here. Should be options.